main.go

```diff
@@ -146,7 +152,7 @@ func main() {
 			disableIAMCollector,
 		},
 		"gke": {
-			collector.NewGKECollector(account, *enableExtendedMetricsGKECollector),
+			collector.NewGKECollector(account, *enableExtendedMetricsGKECollector, *gkeExtraLabelsClusterInfo, *gkeExtraLabelsNodePoolsInfo),
```

Comment:
I think it would be preferable to pass these as a struct|map `gkeExtraLabels` with `ClusterInfo` and `NodePoolsInfo` as fields|keys.